src/main/scala/de/htwg/se/empire/model/player/Player.scala

```diff
@@ -0,0 +1,25 @@
+package de.htwg.se.empire.model.player
+
+import de.htwg.se.empire.model.grid.Country
+import org.apache.logging.log4j.{ LogManager, Logger }
+
+import scala.collection.mutable.ListBuffer
+
+case class Player(name: String, countries: ListBuffer[Country]) {
```

Comment:
Wenn ich darüber nachdenke würde ich fast sagen, dass ein Player die countries in einer lokalen Variable gespeichert haben sollte und nicht im Konstruktor.
- Spieler wird initial ja noch ohne Countries angelegt. Die werden erst vom Controller beim Spielstart zufällig vergeben